test/packages/custom-agent-multiservice/oracle/data_stream/memory/_dev/deploy/agent/custom-agent.yml

```
@@ -0,0 +1,23 @@
+version: '2.3'
+services:
+  docker-custom-agent:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    image: elastic-agent-oracle-client-1
+    depends_on:
+       oracle:
+         condition: service_healthy
+    links:
+      - oracle
+    healthcheck:
+      test: ["CMD", "bash", "-c", "echo 'select 1 from dual;' | ORACLE_HOME=/opt/oracle/instantclient_21_4 /opt/oracle/instantclient_21_4/sqlplus -s sys/Oradoc_db1@oracle:1521/ORCLCDB.localdomain?sysdba=1"]
+      interval: 10s
+      timeout: 10s
+      retries: 60
+    user: root
+  oracle:
+    image: docker.elastic.co/observability-ci/database-enterprise:12.2.0.1
+    ports:
+      - 1521:1521
+      - 5500:5500
```

There is this error in jenkins 
> could not get Docker Compose configuration for service: could not parse port mapping

I think this is caused by these port definitions. You can try by changing them as:

```suggestion
      - 127.0.0.1:1521:1521
      - 127.0.0.1:5500:5500
```

This is the code that looks for the ports: https://github.com/elastic/elastic-package/blob/main/internal/compose/compose.go#L115-L122